### PR TITLE
features/add-mui-lib-to-fe

### PR DESCRIPTION
**Pull request by VladyslavCherednychenko**
add material ui library of react ui components

### CODE REVIEW

**Review by dariia-kachalova — COMMENTED** [review]
Good

**Comment by Tsarar** [comment]
Why just add without using it? :)

**Review by Tsarar — APPROVED** [review]

**Review by Ygoraphael — APPROVED** [review]
Good work

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]
Well done

File: react-portal-employees/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html><html><head><meta charset="utf-8"/><title>Webpack App</title><script defer="defer" src="./main.js"></script><script defer="defer" src="main.js"></script></head><body><h1>Hello world!</h1><h2>Tip: Check your console</h2></body><script>if ('serviceWorker' in navigator) {
+      window.addEventListener('load', () => {
+        navigator.serviceWorker
+          .register('service-worker.js')
+          .then((registration) => {
+            console.log('Service Worker registered: ', registration);
+          })
+          .catch((registrationError) => {
+            console.error('Service Worker registration failed: ', registrationError);
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/dist/index.html]: I think that we need to gitignore this folder

Comment (VladyslavCherednychenko) [reply react-portal-employees/dist/index.html]: Done!

**Pull request merged**